### ISSUE TITLE
Update fastlane

### ISSUE THREAD

**Issue by radko93** [actions/virtual-environments#1163]
**Tool information**
- Tool name: fastlane
- Tool license: not changed
- Add or update? udpdate
- Desired version: latest, at least 2.150.1
- Approximate size: 

**Virtual environments affected**
- [x] macOS 10.15
- [ ] Ubuntu 16.04 LTS
- [ ] Ubuntu 18.04 LTS
- [ ] Ubuntu 20.04 LTS
- [ ] Windows Server 2016 R2
- [ ] Windows Server 2019

**Can this tool be installed during the build?**
This tool is already installed, but we're using an older version.

**Are you willing to submit a PR?**
If that's required then yes.

The new version brings support for new AppStore Connect API.

**Comment by al-cheb** [comment]
Hello, @radko93
The fastlane is updated by automatic and no need any actions. We are planning to finish new rollout by 10/06 Friday with updated fastlane version. I'll let you know as soon as we  finish a deployment. 

Temporary workaround:

```
    steps:
      - name: update fastlane
        run: |
          gem update fastlane
          fastlane --version
```

**Comment by al-cheb** [comment]
@radko93, Fastlane 2.150.1 is now available on macOS 10.15.

**Comment by radko93** [comment]
@al-cheb thank you!
